src/redux/actions/authActions.js

```diff
@@ -1,89 +1,143 @@
-import axios from 'axios';
-import { returnErrors } from './errorActions';
+import {
+  USER_LOADING,
+  USER_LOADED,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  LOGOUT_SUCCESS,
+  REGISTER_SUCCESS,
+  REGISTER_BUSINESS_FAIL,
+  REGISTER_BUSINESS_SUCCESS,
+  REGISTER_FAIL,
+  AUTH_ERROR,
+  OPEN_LOGIN_MODAL,
+  CLOSE_LOGIN_MODAL,
+  OPEN_REGISTER_MODAL,
+  CLOSE_REGISTER_MODAL
+} from "./type";
+import axios from "axios";
+import { returnErrors } from "./errorActions";
 
 export const loaduser = () => (dispatch, getState) => {
   dispatch({ type: USER_LOADING });
-  axios.get('/api/login/user', tokenConfig(getState))
-    .then(res => dispatch({
-      type: USER_LOADED,
-      payload: res.data
-    }))
+  axios
+    .get("/api/login/user", tokenConfig(getState))
+    .then(res =>
+      dispatch({
+        type: USER_LOADED,
+        payload: res.data
+      })
+    )
     .catch(err => {
       dispatch(returnErrors(err.response.data, err.response.status));
       dispatch({
         type: AUTH_ERROR
       });
-    })
-}
+    });
+};
 
 export const register = ({ name, email, password, isVendor }) => dispatch => {
   const config = {
     headers: {
-      'Content-Type': 'application/json'
+      "Content-Type": "application/json"
     }
-  }
+  };
   const body = JSON.stringify({ name, email, password, isVendor });
-  axios.post('/users/signup', body, config)
-    .then(res => dispatch({
-      type: REGISTER_SUCCESS,
-      payload: res.data
-    }))
+  axios
+    .post("/users/signup", body, config)
+    .then(res =>
+      dispatch({
+        type: REGISTER_SUCCESS,
+        payload: res.data
+      })
+    )
     .catch(err => {
-      dispatch(returnErrors(err.response.data, err.response.status, 'REGISTER_FAIL'));
-      dispatch({ type: REGISTER_FAIL })
-    })
-
-}
+      dispatch(
+        returnErrors(err.response.data, err.response.status, "REGISTER_FAIL")
+      );
+      dispatch({ type: REGISTER_FAIL });
+    });
+};
 
 export const login = ({ email, password }) => dispatch => {
   const config = {
     headers: {
-      'Content-Type': 'application/json'
+      "Content-Type": "application/json"
     }
-  }
+  };
   const body = JSON.stringify({ email, password });
-  axios.post('/users/login', body, config)
-    .then(res => dispatch({
-      type: LOGIN_SUCCESS,
-      payload: res.data
-    }))
+  axios
+    .post("/users/login", body, config)
+    .then(res =>
+      dispatch({
+        type: LOGIN_SUCCESS,
+        payload: res.data
+      })
+    )
     .catch(err => {
-      dispatch(returnErrors(err.response.data, err.response.status, 'LOGIN_FAIL'));
-      dispatch({ type: LOGIN_FAIL })
-    })
-}
+      dispatch(
+        returnErrors(err.response.data, err.response.status, "LOGIN_FAIL")
+      );
+      dispatch({ type: LOGIN_FAIL });
+    });
+};
+export const registerBusiness = ({
+  businessname,
+  description,
+  residence,
+  phone
+}) => (dispatch, getState) => {
+  const detail = {
+    description: description,
```

Comment:
when the name and value is same you can just write like it as 
`const detail = {
description,
etc..
}
`